io/io/src/TBufferMerger.cxx

```diff
@@ -40,6 +40,7 @@ void TBufferMerger::Init(std::unique_ptr<TFile> output)
 {
    fFile = output.release();
    fAutoSave = 0;
+   RegisterCallback(nullptr);
```

Comment:
Note that `fCallback` is a reference type, so assigning `nullptr` to it is undefined behaviour.